src/features/forest/components/Tree.tsx

```diff
@@ -82,6 +88,11 @@ export const Tree: React.FC<Props> = ({ treeIndex }) => {
     if (selectedItem !== "Axe") {
       return;
     }
+    const axeAmount = game.context.state.inventory.Axe || new Decimal(0); 
+    if(axeAmount.lessThanOrEqualTo(0))
```

Comment:
Can we tidy up the formatting here so it aligns with the rest of the file. 

```
if (axeAmount.lessThanOrEqualTo(0)) {
    return;
}
```

Reply:
Done ! will do the same for The other PR about minerals